### DISCUSSION TITLE
Welcome new committer:  Zhilong Wang

### DISCUSSION THREAD

**Discussion by zhenjunMa** [mosn/layotto#846] [Announcements]
Hi Community,

On behalf of the Layotto PMC, I am pleased to announce that Zhilong Wang a.k.a @rayowang , has been voted in as Layotto committer.
![image](https://user-images.githubusercontent.com/28104581/204727194-c0c671a7-2a85-4266-8842-3f7818524da1.png)

Zhilong Wang  has been making great contributions to Layotto. Please join me to say congratulations to him!

@rayowang , would you please briefly introduce yourself to the community?

Best Regards.

**Comment by rayowang** [comment]
Hello everybody,  I'm Zhilong Wang, you can also call me rayowang(睿宇), and I love technology since I was a child, so I learned computer science for both undergraduate and master degrees. After graduation, I worked on cloud platform development in Tencent, serverless development in Alibaba, and now work in an e-commerce company and responsible for search microservice governance. 

In the past, it may be because of busy or lazy 😄, I just started open source this year, and very regret not having joined open source community earlier. I have learned a lot in the community during this time.Thanks for the guidance and suggestions of everyone in the community. I hope to continue to study and make progress together with everyone in the future. I also wish the layotto community better and better. Thanks again.

**Reply by seeflood** [reply]
Awesome! Welcome!

**Reply by wenxuwan** [reply]
Respect

**Reply by zhenjunMa** [reply]
Respect! 👏👏（づ￣3￣）づ╭❤～

**Reply by seeflood** [reply]
Are u interested in sharing how alibaba build serverless platform? I'm curious about it 😄
